packages/shared/src/table/selectable-table.tsx

```diff
@@ -51,6 +57,12 @@ const isRowSelected = <T extends K8sResourceCommon>(
   selectedRows: T[]
 ) => selectedRows.some((row) => getUID(row) === rowId);
 
+const canApplyInitialSort = (
+  columns: TableColumnProps[],
+  initialSortIndex: number
+): boolean =>
+  _.isInteger(initialSortIndex) && !!columns[initialSortIndex]?.sortFunction;
```

Comment:
```suggestion
  !!columns[initialSortIndex]?.sortFunction;
```
Shouldn't this be enough?

Reply:
Done.